setup/advanced/ipv6.md

```diff
@@ -0,0 +1,44 @@
+# IPv6
+
+⚠️ IPv6 is an experimental Docker feature, no thorough test was done to ensure there is no IP/DNS leak with Gluetun configured with IPv6. Feel free to create an issue or pull request if you have some testing done and can confirm.
+
+## Setup
+
+1. On your Docker host, edit and create if needed `/etc/docker/daemon.json` with the following JSON key-value pairs:
+
+    ```json
+    {
+      "ipv6": true,
+      "fixed-cidr-v6": "fd00::/80",
+      "experimental": true,
+      "ip6tables": true
+    }
+    ```
+
+    [Reference: Docker documentation on IPv6](https://docs.docker.com/config/daemon/ipv6/#use-ipv6-for-the-default-bridge-network)
+
+    ⁉️ IP range change to unique local address following the note at button of the documentation.
```

Comment:
What do you mean by this?

Reply:
There is a section at the button of the page

![](https://i.imgur.com/fmILGYn.png)

So In my documentation and testing instead of using "2001:db8:1::/64" I used "fd00::/80"

Reply:
Ok I reworded this, hopefully it will clear up things! 👍